setup.py

```diff
@@ -34,12 +33,8 @@
     cmdclass={'build_ext': build_ext},
     version=_VERSION,
     packages=find_packages(),
-    setup_requires=[
-        'numpy',
-    ],

```

Comment:
`setup_requires` is discouraged.
https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-setup-requires